### PR TITLE
Ulitina branch

### PR DESCRIPTION
**Pull request by UlitinaKate**

### CODE REVIEW

File: EkaterinaUlitina/src/main/java/HomeWorkFromLesson6/CopyIntArray.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package HomeWorkFromLesson6;
+
+public class CopyIntArray {
+    public static void main(String[] args) {
+
+        int[][] arr = new int[100][100];
```
Comment (DarynaGrytsaieva) [inline EkaterinaUlitina/src/main/java/HomeWorkFromLesson6/CopyIntArray.java]: Suggest to extract size(100) to separate variable.

File: EkaterinaUlitina/src/main/java/HomeWorkFromLesson6/MaxInDoubleDimensionalArray.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+import java.util.Random;
+
+public class MaxInDoubleDimensionalArray {
+    public static void main(String[] args) {
+
+        int[][] arr = new int[5][8];
+        Random random = new Random();
+
+        int max = 0;
```
Comment (DarynaGrytsaieva) [inline EkaterinaUlitina/src/main/java/HomeWorkFromLesson6/MaxInDoubleDimensionalArray.java]: What if all random numbers are negative and real maximum is -11 for example? Will it work?

File: EkaterinaUlitina/src/main/java/HomeWorkFromLesson6/CopyIntArray.java (hunk 1 above)
Comment (UlitinaKate) [reply EkaterinaUlitina/src/main/java/HomeWorkFromLesson6/CopyIntArray.java]: Done

File: EkaterinaUlitina/src/main/java/HomeWorkFromLesson6/MaxInDoubleDimensionalArray.java (hunk 2 above)
Comment (UlitinaKate) [reply EkaterinaUlitina/src/main/java/HomeWorkFromLesson6/MaxInDoubleDimensionalArray.java]: Fixed it :) 
Thanx!

**Review by DarynaGrytsaieva — APPROVED** [review]

**Pull request merged**